core/arch/arm/plat-ls/conf.mk

```diff
@@ -33,6 +33,7 @@ include core/arch/arm/cpu/cortex-armv8-0.mk
 $(call force,CFG_TEE_CORE_NB_CORE,1)
 $(call force,CFG_DRAM0_SIZE,0x40000000)
 $(call force,CFG_CORE_CLUSTER_SHIFT,2)
+$(call force,CFG_NUM_THREADS,2)
```

Comment:
What is the default value without this change? I guess 1 but I'm not sure, especially since the below line: `CFG_NUM_THREADS ?= CFG_TEE_CORE_NB_CORE` is wrong: missing `$()` -- to be fixed in a separate patch?

I believe regression_1009 expects `CFG_NUM_THREADS >= 2` indeed, but *forcing* 2 might be a bit too strong. How about writing `CFG_NUM_THREADS ?= 2` instead? Then if one wants to build with 3 or 4 or whatever for special use cases, it is easier.

Reply:
Yes, It is set as 1 by default without this change.

_CFG_NUM_THREADS ?= CFG_TEE_CORE_NB_CORE is wrong: missing $() -- to be fixed in a separate patch?_
Yeah, I didn't noticed that, will fix in a separate patch.


_I believe regression_1009 expects CFG_NUM_THREADS >= 2 indeed, but forcing 2 might be a bit too strong. How about writing CFG_NUM_THREADS ?= 2 instead? Then if one wants to build with 3 or 4 or whatever for special use cases, it is easier._
Yes, your point is valid, but what if person gave CFG_NUM_THREADS as 1, this test will again fail.
That's why forced it to 2, but I think we are in a way by setting it as CFG_NUM_THREADS ?= 2 indicating that we need at least 2 threads.
Will do the change.